src/util/Constants.js

```diff
@@ -1483,6 +1514,7 @@ function createEnum(keys) {
  * The type of an {@link ApplicationCommandPermissions} object.
  * @property {Object<ApplicationCommandType, number>} ApplicationCommandTypes
  * The type of an {@link ApplicationCommand} object.
+ * @property {Object{ApplicationRoleConnectionMetadataType, number}} ApplicationRoleConnectionMetadataTypes
```

Comment:
This is malformed.

Reply:
Resolved with 360c2e861b84e24caf1c2e18f0a251168cdd84e6